fastapi/README.md

```diff
@@ -0,0 +1,26 @@
+# SpiceQL FastAPI App
+
+## Create local instance
+
+### 1. Create conda environment
+Create the conda environment to run your local instance in:
+```
+conda create -n spiceql-api -f environment.yaml
+```
+
+### 2. Set environment variables
+Similarly to your SpiceQL conda environment, set `SPICEROOT` or `ISISDATA` to your ISIS data area. You may also need to set `SSPICE_DEBUG` to any value, like `True`.
```

Comment:
Double `SS` for the debug?

Reply:
Yes, I ran into [this error](https://github.com/DOI-USGS/SpiceQL/blob/main/SpiceQL/src/utils.cpp#L1065) once in a different environment I was testing in and then it didn't appear again elsewhere so just wanted to have this down just in case.

Reply:
🐍 👍

Reply:
SSPICE_DEBUG is a hold over from when it was called sugar spice, so it's short for Sugar spice debug. We could update it to SpiceQLDebug but I feel it's kinda whatever.